voluncheer/voluncheer/asgi.py

```diff
@@ -8,16 +8,21 @@
 """
 import os
 
+from channels.auth import AuthMiddlewareStack
 from channels.routing import ProtocolTypeRouter
 from channels.routing import URLRouter
+import django
 from django.core.asgi import get_asgi_application
 
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "voluncheer.settings")
+django.setup()
+
+django_asgi_app = get_asgi_application()
 from chatroom import routing
```

Comment:
Does this import _need_ to be below the above lines? Imports should be at the top of the file.

Reply:
Yes, if I don't get_asgi_application first, then there is no chatroom.